src/Tools/dotnet-gcdump/DotNetHeapDump/DotNetHeapInfo.cs

```diff
@@ -0,0 +1,142 @@
+using FastSerialization;
+using System.Collections.Generic;
+using Address = System.UInt64;
+
+// Copy of version from Microsoft/PerfView
+
+public class DotNetHeapInfo : IFastSerializable
```

Comment:
Classes without namespaces 😢

Reply:
just noticing this as well...  These were copied directly from Microsoft/PerfView.  Nothing says we can't add a namespace for these in our own repo.

Reply:
Seems like I can't put these in namespaces after all... https://github.com/aspnet/AspNetCore/issues/13299#issuecomment-546149821

Looks like it breaks deserialization using the `FastSerializer` format.